Packs/qualys/pack_metadata.json

```diff
@@ -15,6 +15,7 @@
     "keywords": [],
     "marketplaces": [
         "xsoar",
-        "marketplacev2"
+        "marketplacev2",
+        "Xpanse"
```

Comment:
```suggestion
        "xpanse"
```

Reply:
This should fix the [validation error](https://app.circleci.com/pipelines/github/demisto/content/246085/workflows/2e4cd2df-0675-477d-a9f0-1207b9bdce22/jobs/693516) and [unit test / linting errors](https://app.circleci.com/pipelines/github/demisto/content/246085/workflows/2e4cd2df-0675-477d-a9f0-1207b9bdce22/jobs/693516)